.github/workflows/devbuild.yml

```diff
@@ -153,7 +153,7 @@ jobs:
     - name: dependency by pip
       run: |
         sudo pip3 install setuptools
-        sudo pip3 install numpy pytest flake8 jsonschema
+        sudo pip3 install numpy matplotlib pytest flake8 jsonschema
```

Comment:
The added matplotlib is for followup refactor.  It's included with the Qt upgrade so that the other branch of mine does not need to duplicate it.